_data/tools.yml

```diff
@@ -1,30 +1,26 @@
-Alevin: toolshed.g2.bx.psu.edu/repos/bgruening/alevin/alevin
+alevin: toolshed.g2.bx.psu.edu/repos/bgruening/alevin/alevin
 ARAGORN: toolshed.g2.bx.psu.edu/repos/bgruening/trna_prediction/aragorn_trna
 AREsite2: toolshed.g2.bx.psu.edu/repos/rnateam/aresite2/AREsite2_REST
 AddH: toolshed.g2.bx.psu.edu/repos/bgruening/openbabel_addh/openbabel_addh
-BWA-MEM: toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/
-BWA: toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa
+bwa: toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa
 Bandage: toolshed.g2.bx.psu.edu/repos/iuc/bandage/bandage_image
 BlockClust: toolshed.g2.bx.psu.edu/repos/rnateam/blockclust/blockclust
-Bowtie2: toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2
-Bowtie2: toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2
+bowtie2: toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2
 CMV: toolshed.g2.bx.psu.edu/repos/rnateam/cmv/cmv
 Canu: toolshed.g2.bx.psu.edu/repos/bgruening/canu/canu
 CoFold: toolshed.g2.bx.psu.edu/repos/rnateam/cofold/cofold
 Compalignp: toolshed.g2.bx.psu.edu/repos/rnateam/compalignp/compalignp
-Cross Contamination Barcode Filter: toolshed.g2.bx.psu.edu/repos/iuc/crosscontamination_barcode_filter
-DESeq2: toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2
-DESeq2: toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2
+crosscontamination_barcode_filter: toolshed.g2.bx.psu.edu/repos/iuc/crosscontamination_barcode_filter
+deseq2: toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2
 DoRiNA: toolshed.g2.bx.psu.edu/repos/rnateam/dorina/dorina_search
-FastQC: toolshed.g2.bx.psu.edu/repos/iuc/fastqc/fastqc
+fastqc: toolshed.g2.bx.psu.edu/repos/iuc/fastqc/fastqc
```

Comment:
I like those changes, but it can break potentially other sub-sites. Please take care the older ID is not used anymore and replace it in other sites (blog-posts?) as well.

Reply:
ah good point, okay I will leave the old names too and we can just have some duplicates

Reply:
just do a grep in the repo and fix the others ...